packages/studio-ui/src/web/components/Layout/TopNav/cloud/local-storage-pat.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useEffect } from 'react'
+import { fetchPatStatus } from './pat'
+
+const LOCALSTORAGE_KEY = 'bp/pat'
+
+export const LocalStoragePat = (props: { onCompleted: (value?: string) => void }): JSX.Element => {
```

Comment:
Why write a react component if it's to only retrieve pat and never render anything ?

Reply:
Fixed in 62e10bb